src/test/java/Homework22.java

```diff
@@ -0,0 +1,28 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+import org.openqa.selenium.WebElement;
+
+public class Homework22 extends BaseTest{
+
+    @Test
+    public void renamePlaylist() throws InterruptedException {
+
+        String updatedNameMsg = "Updated playlist \"Test Playlist.\"";
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+
+        // step1: login to koel
+        loginPage.login();
+        Thread.sleep(5000);
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.